oncall/autoIssueAssign.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
nit: we should create a `scripts` folder at the root of this repository that this `.sh` file lives in -- we can then rename it to `oncallAutoAssignIssue.sh`

Reply:
This comment still applies -- let's create a `scripts` folder and move this script there, renaming it to `oncallAutoAssignIssue.sh`